### PR TITLE
Portal backoffice frontend

### PR DESCRIPTION
**Pull request by ctf-umbraco**

### CODE REVIEW

**Comment by ctf-umbraco** [comment]
This is only a draft not ready yet

File: Umbraco-StarterKit/Getting-Started/index.md [hunk 1]
```diff
@@ -17,6 +17,25 @@ If you've never worked with Umbraco before, there's a few resources which we rec
 * [How to work with content](link)
 :::
 
+## Portal
+
+The Portal is the landing space for when you enter your project, here you will have the options to manage your project.
+This is also where you can overview your environments, make changes to them, invite users, and Delete environments.
```
Comment (sofietoft) [inline Umbraco-StarterKit/Getting-Started/index.md:23]:
```suggestion
This is also where you can get an overview of your environments, make changes to them, and invite users.
```

File: Umbraco-StarterKit/Getting-Started/index.md [hunk 2]
```diff
@@ -17,6 +17,25 @@ If you've never worked with Umbraco before, there's a few resources which we rec
...
+We will continue to refer to this as the portal throughout the rest of the documentation.
+The portal is a general thing when working with Umbraco on the cloud, it is used on both regular cloud projects and Heartcore as well as Umbraco Uno.
+
+## Backoffice
```
Comment (sofietoft) [inline Umbraco-StarterKit/Getting-Started/index.md:27]:
```suggestion
## Umbraco Backoffice
```

File: Umbraco-StarterKit/Getting-Started/index.md [hunk 3]
```diff
@@ -17,6 +17,25 @@ If you've never worked with Umbraco before, there's a few resources which we rec
...
+
+The Backoffice is where you will be working when editing and creating content, It is divided into sections in the top of the page.
```
Comment (sofietoft) [inline Umbraco-StarterKit/Getting-Started/index.md:29]:
```suggestion
The Umbraco Backoffice is where you will be working when editing and creating content. It is divided into sections in the top of the page.
```

File: Umbraco-StarterKit/Getting-Started/index.md [hunk 4]
```diff
@@ -17,6 +17,25 @@ If you've never worked with Umbraco before, there's a few resources which we rec
...
+In the content section of the backoffice, you will have a content tree on the left side of the screen, This content tree is where your content nodes will be when you start making content.
```
Comment (sofietoft) [inline Umbraco-StarterKit/Getting-Started/index.md:30]:
```suggestion
In the content section of the backoffice, you will have a content tree on the left side of the screen. This content tree is where your content nodes will be when you start making content.
```

File: Umbraco-StarterKit/Getting-Started/index.md [hunk 5]
```diff
@@ -17,6 +17,25 @@ If you've never worked with Umbraco before, there's a few resources which we rec
...
+We will continue to refer to this as the Backoffice throughout the rest of the documentation.
+
+## Frontend
+
+The Frontend is what people see when they visit your website, meaning that this is where all the content you create in the backoffice goes.
+How the frontend looks is completely up to you, and you choose this by creating the look you want in the backoffice, you cannot do any editing on the frontend.
```
Comment (sofietoft) [inline Umbraco-StarterKit/Getting-Started/index.md:36]:
```suggestion
How the frontend looks is completely up to you, and you choose this by creating the look you want in the backoffice. You cannot do any editing on the frontend.
```

Comment (ctf-umbraco) [reply Umbraco-StarterKit/Getting-Started/index.md:36]: I think that is a great idea 👍

**Pull request merged**